src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -573,72 +573,86 @@ query_dfs_mount(const char *path)
 	return idx;
 }
 
-static void
-discover_daos_mount(void)
+/* Discover fuse mount points from /proc/self/mounts and env DAOS_MOUNT_POINT.
```

Comment:
This is just the env now I believe?
```suggestion
/* Discover fuse mount points env DAOS_MOUNT_POINT.
```